src/CalculatorUITestFramework/StandardAoTCalculatorPage.cs

```diff
@@ -0,0 +1,201 @@
+            int Height = getCalculatorWindowSize.Height;
+            if ((Width >= 168) && (Width <= 502) && (Height >= 320) && (Height <= 502))
+            {
+                return;
+            }
+            else
+            {
+                throw new Exception("'Keep on top' window size is too small/large");
+            }
+        }
+        ///// <summary>
+        ///// Gets the AoT ToolTip text
+        ///// </summary>
+        public string GetAoTToolTipText()
+        {
+            string sourceH = this.session.PageSource;
+            if ((sourceH.Contains("Keep on top")) || (sourceH.Contains("Back to full view")))
+            {
+                if (sourceH.Contains("Keep on top"))
+                {
+                    Actions moveToAoTButton = new Actions(WinAppDriver.Instance.CalculatorSession);
+                    moveToAoTButton.MoveToElement(EnterAlwaysOnTopButton);
+                    moveToAoTButton.Perform();
+                }
+                else
+                {
+                    Actions moveToBTFVButton = new Actions(WinAppDriver.Instance.CalculatorSession);
+                    moveToBTFVButton.MoveToElement(ExitAlwaysOnTopButton);
+                    moveToBTFVButton.Perform();
+                }
+            }
+            else
+            {
+                throw new NotFoundException("Could not find 'Keep on top' button or 'Back to full view' button");
+            }
+            var ToolTipText = ToolTip.Text.ToString();
+            return ToolTipText;
+        }
+        ///// <summary>
+        ///// Checks in AoT (Keep on top) button is present
+        ///// </summary>
+        public string GetAoTPresence()
+        {
+            bool AoTPresent;
+            string sourceJ = this.session.PageSource;
+            if (sourceJ.Contains("Keep on top"))
+            {
+                AoTPresent = true;
+            }
+            else
+            {
+                AoTPresent = false;
+            }
+            return AoTPresent.ToString();
+        }
+        ///// <summary>
+        ///// Checks Standard calculator to see if it's in AoT (Keep on top)
+        ///// </summary>
+        public string AoTModeCheck()
+        {
+            bool InAoTMode;
+            string sourceK = this.session.PageSource;
+            if ((sourceK.Contains("Keep on top")) && (sourceK.Contains("Header")))
+            {
+                InAoTMode = false;
+            }
+            else
+            {
+                InAoTMode = true;
+            }
+            return InAoTMode.ToString();
+        }
+        /// <summary>
+        /// Checks to see if the size of the Calculator window is within the AoT arange of size
+        /// To-do: Find a way to get resolution and then do something like monitor quadrant location, or
+        /// something like (X >= (0.8*screenWidth))... etc.
+        /// </summary>
+        public void AoTWindowPositionWithinRange()
```

Comment:
This is testing the compact overlay api

Reply:
This is test framework method that helps to verify that the calculator is in AoT mode.